.github/workflows/build.yml

```diff
@@ -122,6 +122,8 @@ jobs:
     steps:
       - name: Checkout OpenFPGA repo
         uses: actions/checkout@v2
+        with:
+            submodules: true
```

Comment:
Since the `make all` is used in compilation guideline, should we update the build script in CI? It will avoid confuse users.

Reply:
I don't think so. CI is working well right now and it is not really necessary to have CI commands aligned with end-user compilation/installation commands. I think we are good to now with this checkin. I added Makefile more as a convenience to the end-user and not necessarily required for replacing yosys. If you strongly think that CI should be aligned then I would rather remove Makefile which I added and let flow be the same as it was earlier.